charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -75,6 +32,7 @@ spec:
         prometheus.io/scrape: "{{ .Values.global.prometheus.enabled }}"
         prometheus.io/port: "{{ .Values.global.prometheus.port }}"
         prometheus.io/path: "/"
+        dapr.io/app-id: dapr-operator
```

Comment:
I would not set app-id in a control plane service. We use these for sidecars only

Reply:
Control plane components will now all also receive SVIDs, and have SPIFFE IDs in the Dapr trust domain scheme. The `dapr.io/app-id` annotation is the mechanism which the Kubernetes sentry validator uses to check the requested app ID. This means that the annotation is now required for control plane components.

Reply:
Should we use a different annotation for control-plane components however? Maintainers may have an opinion

Reply:
That would be fine by me.

Reply:
We need to document which app-ids are reserved. I would even claim that "dapr-" prefix is reserved for internal services.